src/screens/TopicPageScreen/index.js

```diff
@@ -428,10 +395,10 @@ const TopicPageScreen = (props) => {
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
       <NavHeader
         domain={topicName}
-        animatedHeight={animatedHeight}
+        animatedHeight={headerHeight}
         onShareCommunity={onShareCommunity}
         isHeaderHide={isHeaderHide}
-        opacityHeaderAnimation={opacityHeaderAnimation}
+        opacityHeaderAnimation={opacityHeader}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The props passed to `NavHeader` have been updated. Ensure that these changes are reflected in the `NavHeader` component and do not break its functionality.
```diff
-        animatedHeight={animatedHeight}
+        animatedHeight={headerHeight}
         onShareCommunity={onShareCommunity}
         isHeaderHide={isHeaderHide}
-        opacityHeaderAnimation={opacityHeaderAnimation}
+        opacityHeaderAnimation={opacityHeader}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->